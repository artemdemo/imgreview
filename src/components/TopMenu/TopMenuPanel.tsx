import React from 'react';
import styled from 'styled-components';
import ClearFix from '../Floating/ClearFix';
import * as styleVars from '../../styles/variables';

const TopMenuPanel = styled(ClearFix)`
    padding: 4px 15px;
    background-color: ${styleVars.mainMenuColor};
    z-index: ${styleVars.mainMenuZIndex};
`;

export default TopMenuPanel;
