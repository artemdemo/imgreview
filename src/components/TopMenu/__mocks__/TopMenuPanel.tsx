import React from 'react';

const TopMenuPanel = props => (
    <div data-mock='TopMenuPanel'>
        {props.children}
    </div>
);

export default TopMenuPanel;
