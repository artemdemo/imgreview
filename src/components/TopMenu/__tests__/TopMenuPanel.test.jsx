import React from 'react';
import renderer from 'react-test-renderer';
import TopMenuPanel from '../TopMenuPanel';

describe('TopMenuPanel', () => {
  it('simple render', () => {
    const tree = renderer.create(<TopMenuPanel />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
