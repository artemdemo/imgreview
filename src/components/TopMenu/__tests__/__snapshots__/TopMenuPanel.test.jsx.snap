// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopMenuPanel simple render 1`] = `
<div
  className="sc-bdVaJa sc-bwzfXH QZZFO"
/>
`;
